src/plugins/formvalid/formvalid-fr.hbs

```diff
@@ -448,6 +428,13 @@
 			<input type="submit" value="Soumettre" class="btn btn-primary" /> <input type="reset" value="Réinitialiser la page aux valeurs par défaut" class="btn btn-link btn-sm show p-0 pt-4" />
 		</form>
 	</div>
+</section>
+<hr>
+<section>
+<!-- checkbox in list item -->
+	<h2>Exemples de formulaires empilés et horizontaux utilisant des éléments de liste comme contenants</h2>
```

Comment:
I improvised this heading's content.

Reply:
Instead of "empilés" you can say "formulaires verticaux et horizontaux"

Reply:
Done. Also applied it to other instances of "empilés".